packagename/__init__.py

```diff
@@ -6,16 +6,20 @@
 from ._astropy_init import *
 # ----------------------------------------------------------------------------
 
-# Uncomment to enforce Python version check during package import.
+# Enforce Python version check during package import.
 # This is the same check as the one at the top of setup.py
-#import sys
-#class UnsupportedPythonError(Exception):
-#    pass
-#__minimum_python_version__ = '3.5'
-#if sys.version_info < tuple((int(val) for val in __minimum_python_version__.split('.'))):
-#    raise UnsupportedPythonError("{} does not support Python < {}".format(__package__, __minimum_python_version__))
+import sys
+
+__minimum_python_version__ = "2.7"
```

Comment:
@Cadair - When did we changed it back? This should have remained 3.5 or even 3.6, shouldn't it?

Reply:
I changed it to 2.7 so that the whole travis build passed. I suggest we change it back as part of #319

Reply:
Thinking about this a little more, I think we should keep this as 2.7 for as long as the template supports 2.7 so that we are testing the rendered version of the template with all the supported versions of Python.

Reply:
I think the rendered version should indeed be 2.7 for now